token/core/zkatdlog/nogh/v1/validator/testdata/generate.go

```diff
@@ -0,0 +1,10 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package testdata
+
+//go:generate idemixgen ca-keygen --output ./bls12_381_bbs/ca --curve BLS12_381_BBS --aries
+//go:generate idemixgen signerconfig --ca-input ./bls12_381_bbs/ca --output ./bls12_381_bbs/idemix --admin -u example.com -e alice -r 150 --curve BLS12_381_BBS --aries
```

Comment:
I am wondering if the auditor and the validator could share the generated testdata?